simple_use_case/pydoit/dodo.py

```diff
@@ -53,28 +55,79 @@ def task_plot_over_line():
     pol = ROOT / "plotoverline.csv"
     return {
         "file_dep": [pvdfile, vtufile, postproc],
-        "actions": [f"pvbatch {postproc} {pvdfile} {pol}"],
+        "actions": [f"pvbatch {postproc} -i {pvdfile} -o {pol}"],
         "targets": [pol],
         "clean": True,
     }
 
 
-def task_copy_paper():
-    """copy paper.tex from source directory"""
-    source_code = SOURCE / "paper.tex"
-    root_code = ROOT / "paper.tex"
+def task_write_paper_source():
+    """write paper source to file"""
+
+    def write_source(gmsh_stdout, dependencies, targets):
+        # file names in correct order
+        fnames = ["lineplot.tex", "plotoverline.csv", "numdofs.txt"]
+        deps = fix_deps(dependencies, fnames)
+        preamble = """\\documentclass[12pt]{article}
```

Comment:
Instead of writing the complete paper to a file (which does not allow to compile this), I would rather "\input" or "\include" only the macro definition that is then written in this task". This way, someone just interested in compiling the paper manually (while changing the "raw" text in his/her text editor can do so (otherwise you would always have to call the dodo task write_paper_source). In addition, this allows you to work in a standard latex editor that understands the syntax, ..

Reply:
so I changed this in the python-action as desired.